### DISCUSSION TITLE
Issue with GitHub Enterprise for Education Access

### DISCUSSION THREAD

**Discussion by atosh502** [community/community#142539] [GitHub Education] [answered]
### Hi everyone,

Hi everyone,

I’m an instructor for a university course where we use GitHub Classroom to manage labs and assignments. We’ve reached the limit of the free plan and decided to upgrade to GitHub Enterprise Cloud for Education.

We tried [signing up](https://education.github.com/discount_requests/application) for [GitHub Enterprise for Education](https://github.com/education/schools) but were informed that our organization already has access, and an existing admin is managing the account. GitHub support has forwarded our request to the admin multiple times over the past two weeks, but we still haven’t received a response, causing a delay.

Has anyone been able to resolve a similar issue? Any help would be greatly appreciated.

Thank you, 
Aashutosh

**Comment by samus-aran** [comment]
Hi @atosh502 👋 

Thank you for your patience. The community is unable to assist with nor escalate your ticket and you are more likely to get a useful response if you are posting your question(s) in the applicable category. I've gone ahead and moved it for you. 

Whilst the support team assist you with your Enterprise request can we ask what limitations have you reached for your current Teams benefits. I can see you are a verified student, as a verified educator you would be able to utilise the educators' benefits.

**Reply by atosh502** [reply, marked as answer]
Hi @samus-aran, thanks for the reply. 

We found the admin for our school's GitHub Enterprise for Education. So the issue is not relevant anymore. We were reaching the CI/CD limits in our free organization (+classroom) and wanted to upgrade to the enterprise education cloud.

Thank you, 
Aashutosh

**Reply by samus-aran** [reply]
Fabulous for responding! I'm glad you were able to get it sorted ✨
